### PR TITLE
Overrides Bootstrap button colors, updates button class names to match, ...

### PR DESCRIPTION
**Pull request by mtribone**
...variables for style sheet. Eliminating the need for additional overrides in ScholarSphere.

### CODE REVIEW

File: app/views/batch_edits/_delete_selected.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%= form_tag(batch_edits_path, method: :delete, class: "batch-select-all hidden", "data-behavior" => 'batch-select-all') do -%>
     <%= hidden_field_tag('update_type', 'delete_all') %>
     <%= hidden_field_tag('return_controller', params[:controller]) %>
-  <%= submit_tag("Delete Selected", class: 'batch-all-button btn btn-primary submits-batches', data: { confirm: "Deleting a file from #{t('sufia.product_name')} is permanent. Click OK to delete this file from #{t('sufia.product_name')}, or Cancel to cancel this operation" }) %>
+  <%= submit_tag("Delete Selected", class: 'batch-all-button btn btn-danger submits-batches', data: { confirm: "Deleting a file from #{t('sufia.product_name')} is permanent. Click OK to delete this file from #{t('sufia.product_name')}, or Cancel to cancel this operation" }) %>
```
Comment (mjgiarlo) [inline app/views/batch_edits/_delete_selected.html.erb]: DAAAAAAAAANGER :skull: :no_entry_sign: :bomb: :hocho: :gun:

**Comment by mjgiarlo** [comment]
This looks good to me!  I might like to see, if it's not a huge pain, before and after screenshots that give folks an idea what the changes do to the UI.  Other than that, I am :+1: on this but defer to someone not from Penn State to merge.

**Comment by mtribone** [comment]
@mjgiarlo You got it. I'll have them in a jiffy.

**Comment by mtribone** [comment]
Current Sufia

screen shot 2015-02-04 at 3 21 20 pm
screen shot 2015-02-04 at 3 21 42 pm
screen shot 2015-02-04 at 3 22 04 pm
screen shot 2015-02-04 at 3 22 38 pm

**Comment by mtribone** [comment]
Updated Sufia

screen shot 2015-02-04 at 3 16 30 pm
screen shot 2015-02-04 at 3 17 22 pm
screen shot 2015-02-04 at 3 17 52 pm
screen shot 2015-02-04 at 3 18 49 pm

**Comment by mjgiarlo** [comment]
:clap:

**Pull request merged**